priv/repo/migrations/20180315082449_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule App.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :login, :string
+      add :user_id, :integer
```

Comment:
just checking, how are you planning to be using this `user_id`? Could you just use the primary key `id` automatically generated here by [`create table(:users)`](https://hexdocs.pm/ecto/Ecto.Migration.html#table/2)

Reply:
yes I think it could be the primary key. I will need to figure out the ecto changes to do this :+1:

Reply:
issue created for using Github ids as primary keys: https://github.com/dwyl/github-backup/issues/101